clients/src/main/java/org/apache/kafka/common/security/authenticator/SaslServerAuthenticator.java

```diff
@@ -104,14 +106,16 @@
     private NetworkReceive netInBuffer;
     private Send netOutBuffer;
 
-    public SaslServerAuthenticator(String node, JaasContext jaasContext, final Subject subject, KerberosShortNamer kerberosNameParser, String host, int maxReceiveSize, CredentialCache credentialCache) throws IOException {
+    public SaslServerAuthenticator(String node, JaasContext jaasContext, final Subject subject,
+                                   KerberosShortNamer kerberosNameParser, String host, int maxReceiveSize,
```

Comment:
Fair point. I'm inclined to remove it since the maximum configured receive is unlikely to be less than what we use here.